mustermann/lib/mustermann/ast/expander.rb

```diff
@@ -124,6 +124,8 @@ def error_for(values)
       # @see Mustermann::AST::Translator#expand
       # @!visibility private
       ruby2_keywords def escape(string, *args)
+        return super unless string.respond_to?(:=~)
```

Comment:
If it's just nil/false, I'd simplify to `return super unless string` or integrate it in the ternary below.
Or are other types seen?

Reply:
From the build failures, it's `Integer`s: https://github.com/magni-/mustermann/runs/7425121555?check_suite_focus=true#step:5:14

e.g.
```
   9) Mustermann::Expander supports combining different pattern styles
     Failure/Error: expander.expand(bar: 23).should be == "/23"
     NoMethodError:
       undefined method `=~' for 23:Integer
     # ./mustermann/lib/mustermann/ast/expander.rb:128:in `escape'
     # ./mustermann/lib/mustermann/ast/expander.rb:99:in `block in expand'
     # ./mustermann/lib/mustermann/ast/expander.rb:99:in `each'
     # ./mustermann/lib/mustermann/ast/expander.rb:99:in `expand'
     # ./mustermann/lib/mustermann/expander.rb:150:in `expand'
     # ./mustermann/spec/expander_spec.rb:26:in `block (2 levels) in <top (required)>'
```